SpatialGDK/Source/Public/Schema/StandardLibrary.h

```diff
@@ -6,6 +6,7 @@
 #include "Schema/Component.h"
 #include "SpatialConstants.h"
 #include "UObject/UObjectGlobals.h"
+#include "UObject/Package.h"
```

Comment:
If you happen to touch this PR again for another reason, please fix the alphabetical ordering. If not no worries not worth it.